src/core/src/SettingsWrapper.cpp

```diff
@@ -12,6 +12,7 @@ const std::string_view SESSION_ION_SERVER_URL_BASE =
     "/persistent/exts/cesium.omniverse/sessions/session{}/ionServerUrl";
 const std::string_view SESSION_USER_ACCESS_TOKEN_BASE =
     "/persistent/exts/cesium.omniverse/sessions/session{}/userAccessToken";
+const char* MAX_CACHE_ITEMS_PATH = "/persistent/exts/cesium.omniverse/maxCacheItems";
```

Comment:
It's passed directly to the carb::settings interface, which takes const char * values. The other settings go through fmt first. So const char * seems to me like the natural type for it; happy to change it if you feel otherwise.